packages/components/src/internal/components/domainproperties/list/ListPropertiesAdvancedSettings.tsx

```diff
@@ -416,10 +421,23 @@ export class AdvancedSettings extends React.PureComponent<AdvancedSettingsProps,
 
         this.state = {
             modalOpen: false,
+            objectLevelDiscussionsEnabled: false,
             ...initialState,
         } as AdvancedSettingsState;
     }
 
+    componentDidMount() {
```

Comment:
Waffling on the best way to go about this. Because this deprecated feature will be gone soon, I am a little tempted to add a temporary prop for test purposes that triggers whether or not this call will run. We seem to not have had the need up until now to check whether a feature is deprecated on the client-side.

Reply:
See my comment in the platform PR regarding putting this deprecated feature flag onto the LABKEY page context JSON object so that we don't have to do this API call to get the value. If we do that, then the test would just need to set the value directly in that LABKEY object (see example in `NameIdSettings.test.tsx` for how it sets something in `LABKEY.moduleContext` before the test case)